types/jquery.soap/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for jQuery.SOAP 1.6.7
+// Type definitions for jQuery.SOAP 1.7.3
```

Comment:
This version is not in the right format. It should just be 1.7.
Could you remove the line in the `tslint.json` file that says `dt-header: false` and fix any other errors that come up?
If you want, you can remove any other such lines from the `tslint.json`, but that might be a lot of work.
Thanks!

Reply:
hey @uniqueiniquity Just removed the dt-header: false and fixed the version number.```npm run lint jquery.soap```  worked.